### PR TITLE
Refactor the utah tax rate yaml file to align with the code standards

### LINKED ISSUE #4023: Refactor the utah tax rate yaml file to align with the code standards
Change the unit to "/1"
Remove the "value" inputs 
Add a period

### PR DESCRIPTION
**Pull request by laviniawo**
Fixes #4023

### CODE REVIEW

File: policyengine_us/parameters/gov/states/ut/tax/income/rate.yaml [hunk 1]
```diff
@@ -1,15 +1,11 @@
 description: Utah main income tax rate.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ut/tax/income/rate.yaml:1]:
```suggestion
description: Utah taxes personal income at this flat rate. 
```

File: policyengine_us/parameters/gov/states/ut/tax/income/rate.yaml [hunk 2]
```diff
@@ -1,15 +1,11 @@
...
-  2023-01-01:
-    value: 0.0465
+  2008-01-01: 0.05
+  2018-05-08: 0.0495
+  2022-01-01: 0.0485
+  2023-01-01: 0.0465
 metadata:
-  unit: bool
+  unit: /1
   label: Utah income tax rate
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ut/tax/income/rate.yaml:9]:
```suggestion
  label: Utah income tax rate
  period: year
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

File: changelog_entry.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    fixes:
+    - Refactor the utah tax rate yaml file.
```
Comment (MaxGhenis) [inline changelog_entry.yaml:4]:
```suggestion
    - Refactor the Utah tax rate yaml file.
```

File: changelog_entry.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
```
Comment (MaxGhenis) [inline changelog_entry.yaml:1]: As it doesn't introduce new functionality
```suggestion
- bump: patch
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**